manifests/init.pp

```diff
@@ -28,7 +28,8 @@
 #
 # [*bind_address*]
 #   The ip address we are going to bind the corosync daemon too.
-#   Can be specified as an array to have multiple rings.
+#   Can be specified as an array to have multiple rings or can have the value
+#   'UNSET' in order to remove the `bindnetaddr` corosync directive.
```

Comment:
We moved from UNSET to undef recently, see #322. Can you update your PR accordingly?

Reply:
Hmm

It looks like 
```puppet
class { 'corosync':
  bind_address => undef,
  ...
}
```
Results in having `$bind_address` set to the default found in `$corosync::params::bind_address` which is `$::facts[ipaddress]`.

I'm not sure what would be the best to allow an `undef` value here while still having default to `$::facts[ipaddress]`. Could you give me a hint please ?

Regards,

Reply:
More precisely, this is related to [PUPPET-5295](https://tickets.puppetlabs.com/browse/PUP-5295). Maybe we could switch to `$bind_address` being an empty array when we want to remove `bindnetaddr` from corosync.conf ?

Reply:
IMHO we should make deeper changes around this useless `$bind_address` parameter when `$quorum_members` is defined (see my comment in #422). We should remove it from `corosync.conf`.
I will look at this in the next days, unless you have some proposal.